.gitignore

```diff
@@ -1,5 +1,6 @@
 .phpunit.result.cache
 /vendor
+/tests/fixtures
```

Comment:
We need to remove this line, rerun PHP VCR and then commit the new fixtures so that we have them in the Repo.